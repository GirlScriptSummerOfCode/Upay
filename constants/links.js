export default [{ title: 'Upay', url: '/' }, { title: 'About', url: '/about' }];
