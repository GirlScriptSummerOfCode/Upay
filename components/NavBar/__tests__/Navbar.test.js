/* globals describe, it, expect */
import React from 'react';
import { mount } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json'; //removes unnecessary JSON from snapshot
import Navbar from '../';
import links from '../../../constants/links';

describe('<Navbar />', () => {
  it('renders correctly', () => {
    const wrapper = mount(<Navbar links={links} />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
