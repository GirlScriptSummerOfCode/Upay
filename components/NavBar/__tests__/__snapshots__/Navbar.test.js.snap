// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> renders correctly 1`] = `
<NavBar
  links={
    Array [
      Object {
        "title": "Upay",
        "url": "/",
      },
      Object {
        "title": "About",
        "url": "/about",
      },
    ]
  }
>
  <JSXStyle
    css="nav.jsx-2694676140{background:linear-gradient(90deg,violet,limegreen);}nav.jsx-2694676140 ul.jsx-2694676140{margin:0;padding:20px;list-style:none;}nav.jsx-2694676140 ul.jsx-2694676140 li.jsx-2694676140{display:inline-block;margin:5px;}nav.jsx-2694676140 ul.jsx-2694676140 li.jsx-2694676140 a.jsx-2694676140{color:white;-webkit-text-decoration:none;text-decoration:none;}"
    styleId="2694676140"
  />
  <nav
    className="jsx-2694676140"
  >
    <ul
      className="jsx-2694676140"
    >
      <li
        className="jsx-2694676140"
        key="0"
      >
        <Link
          href="/"
        >
          <a
            className="jsx-2694676140"
            href="/"
            onClick={[Function]}
          >
            Upay
          </a>
        </Link>
      </li>
      <li
        className="jsx-2694676140"
        key="1"
      >
        <Link
          href="/about"
        >
          <a
            className="jsx-2694676140"
            href="/about"
            onClick={[Function]}
          >
            About
          </a>
        </Link>
      </li>
    </ul>
  </nav>
</NavBar>
`;
